### PR TITLE
Delete magic variables

### PR DESCRIPTION
**Pull request by weihubeats**
Fixes https://github.com/apache/incubator-eventmesh/issues/2310

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java [hunk 1]
```diff
@@ -112,15 +112,15 @@ public void testSubscribeWithUrl() {
     @Test
     public void testSubscribeStreamWithoutListener() {
         eventMeshGrpcConsumer.subscribe(Collections.singletonList(buildMockSubscriptionItem()));
-        verifyZeroInteractions(consumerAsyncClient);
+        verifyNoMoreInteractions(consumerAsyncClient);
```
Comment (jonyangx) [inline eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java:115]: I figure it is better use verifyZeroInteractions method here.

Comment (weihubeats) [reply eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java:115]: verifyZeroInteractions is a deprecated method that also calls verifyNoMoreInteractions at the bottom.
image

Comment (jonyangx) [reply eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java:115]: ok

**Review by jonyangx — APPROVED** [review]

**Comment by xwm1992** [comment]
@weihubeats please solve the conflicts

**Comment by weihubeats** [comment]
> @weihubeats please solve the conflicts

done

**Review by jonyangx — APPROVED** [review]

**Pull request merged**